chemprop/args.py

```diff
@@ -770,11 +996,13 @@ class PredictArgs(CommonArgs):
     """Location to save the results of uncertainty evaluations."""
     uncertainty_dropout_p: float = 0.1
     """The probability to use for Monte Carlo dropout uncertainty estimation."""
+    alpha: float = 0.1
```

Comment:
This argument has the same name as the one in the training args section. Should have distinct names to avoid weird overwriting edge cases at the very least.

Is this one just being used for the calibration operations?

Reply:
Yes, this value of alpha is being used in the calibration operations. Perhaps this one can be renamed to conformal_alpha and the other one can be quantile_loss_alpha?

Reply:
Those seem like good names

Reply:
Just added these changes.